versionmanager/manager.go

```diff
@@ -112,12 +114,22 @@ func (m VersionManager) Evaluate(requestedVersion string, proxyCall bool) (strin
 			}
 		}
 
+		m.conf.Displayer.Display("No compatible version found locally, search a remote one...")
 		if m.conf.NoInstall {
 			m.conf.Displayer.Flush(proxyCall)
```

Comment:
This line is not needed here anymore, however it should be called before `return "", errNoCompatibleLocally`